style.css

```diff
@@ -129,6 +129,14 @@ button {
   padding: 0 12px;
 }
 
+.vertical-bar{
+  width: 4px;
+  height: 20px;
+  background: #c0c0c0;
+  box-shadow: inset -1px -1px var(--window-frame), inset 1px 1px var(--button-highlight), inset -1px -1px var(--button-highlight), inset 1px 1px var(--window-frame);
```

Comment:
Can we build this out of `var(--border-raised-outer), var(--border-raised-inner);` or something similar?

Reply:
Yes, that would make more sense. I haven't done very much with CSS variables before, so what's the syntax?

Reply:
Give the box-shadow for `button` a peek (a few lines above)